content/en/security/cspm/_index.md

```diff
@@ -11,18 +11,47 @@ Cloud Security Posture Management is not currently available in this site.
+By continuously surfacing security weaknesses resulting from misconfigurations, teams can easily mitigate risks while ensuring compliance with industry standards.
 
-{{< img src="security/cspm/landing_page.png" alt="Cloud Security Posture Management" width="100%">}}
+## Detect misconfigurations across your cloud resources
 
-Assess the configuration of your cloud resources, such as security groups, storage buckets, load balancers, and databases against configuration rules. Use the Datadog Agent to review local configuration information from servers, containers, and Kubernetes clusters against Datadog's OOTB Posture Management [Cloud][1] and [Infrastructure][2] detection rules.
+Strengthen your security posture and achieve continuous compliance by detecting, prioritizing, and remediating misconfigurations across all your cloud resources using Datadog's [out-of-the-box detection rules](#manage-out-of-the-box-and-custom-detection-rules). 
 
-View your cloud security posture at a high level with the [Posture Management][3] page. Examine the details of findings and analyze historical configurations with [Findings][4].
+View a high-level overview of your security posture on the [Overview page][3]. Examine the details of findings and analyze historical configurations with the [Security Findings Explorer][9].
 
-## Glossary
+{{< img src="security/cspm/overview_page.png" alt="Cloud Security Posture Management overview page" width="100%">}}
+
+## Maintain compliance with industry frameworks and benchmarks
+
+CSPM comes with a set of 400+ out-of-the-box detection rules which are maintained by a team of security experts. Each rule maps to one or more controls and requirements within a compliance standard or industry benchmark, such as the popular PCI and SOC2 compliance frameworks.
+
+[View compliance reports][10] that allow you to see how well you're doing against each control in a compliance framework. The reports include details such as resources with the most failed findings, a comprehensive breakdown of the number of resources with pass/fail findings, and the top three high-severity rule failures.
+
+{{< img src="security/cspm/compliance_frameworks.png" alt="Cloud Security Posture Management compliance frameworks" width="100%">}}
+
+## Manage out-of-the-box and custom detection rules
+
+[Out-of-the-box detection rules][7] surface the most important risks so that you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account. Customize the rules by defining [how each rule scans your environment][6], [create custom rules][8] (GCP only), and [set up real-time notifications for failed findings](#set-up-real-time-notifications).
```

Comment:
```suggestion
[Out-of-the-box detection rules][7] surface the most important risks so that you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account. [Customize the rules][6] by defining how each rule scans your environment, [create custom rules][8] (GCP only), and [set up real-time notifications for failed findings](#set-up-real-time-notifications).
```